lib/doofinder_api_items.php

```diff
@@ -55,6 +60,24 @@ public function deleteBulk($payload)
         return $this->post($url, $payload);
     }
 
+    /**
+     * Make a request to the API to reprocess all the feeds
+     *
+     * @param array items ids
+     */
+    public function invokeReindexing()
```

Comment:
This class (DoofinderApiItems) is for update on save. Please put it in another place. If necessary create a new class (For example: DoofinderApiIndex).

Reply:
Created new class on: https://github.com/doofinder/doofinder-prestashop/pull/143/commits/13a4e4631a3fff59967320600ca685099550cd59